pre-cache/check_space

```diff
@@ -11,8 +11,25 @@ validate_disk_space() {
     SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
     TOKEN=$(cat ${SERVICEACCOUNT}/token)
     CACERT=${SERVICEACCOUNT}/ca.crt
-    high=$(curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -X GET ${APISERVER}/api/v1/nodes/${NODE_NAME}/proxy/configz | \
+    
+    retries=0
+    max_retries=3
+    while [[ $retries -lt $max_retries ]]; do
+      high=$(curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -X GET ${APISERVER}/api/v1/nodes/${NODE_NAME}/proxy/configz | \
                 chroot /host jq '.kubeletconfig.imageGCHighThresholdPercent')
+      if [[ $high -gt 0 && $high -lt 101 ]]; then
+         break
+      else
+         ((retries=retries+1))
+         if [[ $retries -eq $max_retries ]]; then
+            log_info "Could not get imageGCHighThresholdPercent value from kubelet. Using default value instead."
+            high=85
+         else
+            sleep 5
+        fi
+      fi
+    done
```

Comment:
I think it will be beneficial to define a `retry` function in `pre-cache/common` and reference the function here instead. You can then proceed to validate the `high` value after the `retry` function call. The `retry` function is quite a useful utility function and may easily be used in other cases as well. 
This suggestion follows the same procedure applied in `upgrade-recovery.sh` (https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/main/recovery/bindata/upgrade-recovery.sh#L64-L93).